### PR TITLE
MGMT-4937 Events doc

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

File: docs/events.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Using events, a user should be able to understand how the cluster reached its current state, and understand what, if anything, is wrong with it.
+
+## Event emission guidelines
+
+### ERROR
+Something wrong has happened that must be investigated.  These should be relatively rare and are important to understand, so make them as verbose as possible - describe what happened, what the user might do to mitigate it, and anything needed to debug it.
+
+Examples:
+1. REST API call failure.
```
Comment (osherdp) [inline docs/events.md:13]: I think it's weird listing only one item. I would have add some more examples, or put it in a sentence

File: docs/events.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Events
+
+Events that may be of interest to users of the Assisted Service are made available via the REST API.  The events are stored in the SQL database.  Each event is associated with a cluster and has a severity; some events are also associated with a host.
+
+Using events, a user should be able to understand how the cluster reached its current state, and understand what, if anything, is wrong with it.
+
+## Event emission guidelines
+
+### ERROR
+Something wrong has happened that must be investigated.  These should be relatively rare and are important to understand, so make them as verbose as possible - describe what happened, what the user might do to mitigate it, and anything needed to debug it.
```
Comment (osherdp) [inline docs/events.md:10]: Maybe it should be good to say "be verbose, but without exposing user secrets". Not necessarily here but somewhere in this doc

**Review by osherdp — CHANGES_REQUESTED** [review]

File: docs/events.md (hunk 2 above)
Comment (avishayt) [reply docs/events.md:12]: Added a note

File: docs/events.md (hunk 1 above)
Comment (avishayt) [reply docs/events.md:15]: Added more examples

**Review by osherdp — APPROVED** [review]

**Pull request merged**